s3fs/core.py

```diff
@@ -71,6 +71,59 @@ def split_path(path):
              'authenticated-read'}
 
 
+def title_case(string):
+    """
+    TitleCases a given string.
+
+    Parameters
+    ----------
+    string : underscore seperated string
+    """
+    return ''.join([x.capitalize() for x in string.split('_')])
+
+
+class SSEParams(object):
```

Comment:
How about, instead of a class, a simple function with keyword arguments like the __init__ here, that gives you back a dictionary to be used as kwargs?